src/view/components/AlphaPickerContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
I think these should go under `/src/components` somewhere. Maybe have an `itemsList` or `itemsView` or just `items` subfolder there?

Reply:
i am thinking like this  /src/components/common/

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/d7e48d30b6480d3af9083b3aa36eebb4331a2d3f